special-use/special-use-POST-suds-mapping-post.json

```diff
@@ -86,6 +77,21 @@
       "temp-outfitter-fields": {
         "type": "object",
         "properties": {
+          "individualIsCitizen" : { "type" : "boolean", "note" : "Required if orgType is individual. Stored in the middle layer."},
```

Comment:
Let's convert the "Stored in a middlelayer" to a flag of "stored-in-middle-layer": true